### DISCUSSION TITLE
To grant administrative access to a collaborator

### DISCUSSION THREAD

**Discussion by sudhamrit** [community/community#58436] [Repositories]
### Select Topic Area

General

### Feature Area

Issues

### Body

Hi, 
I want grant administrative access to a collaborator on our private GitHub repository, but there is no option to make "Admin" or "Write" access to grant administrative privileges.

How can i do.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by airtower-luna** [comment]
Is that repository owned by a personal account or an organization? AFAIK for a personal repository the owner is the admin and that's it, more complex management is only available for organizations.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sudhamrit ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
